Sources/SnapshotTesting/Strategy/CALayer.swift

```diff
@@ -28,16 +28,30 @@ import UIKit
 
 extension Strategy where Snapshottable == CALayer, Format == UIImage {
   public static var image: Strategy {
-    return .image(precision: 1)
```

Comment:
Rather than maintain a jumble of overloads, paring it down to one property and one function with all defaults.